src/util/downloadUrlByOS.ts

```diff
@@ -15,14 +15,7 @@ export default function downloadUrlByOS(
   }
 
   if (userOS === UserOS.WIN) {
-    if (
-      navigator.appVersion.includes('WOW64') ||
-      navigator.appVersion.includes('Win64')
-    ) {
-      return `${baseURL}/node-${version}-x64.msi`;
-    }
-
-    return `${baseURL}/node-${version}-x86.msi`;
+    return `${baseURL}/node-${version}-x64.msi`;
```

Comment:
Wouldn't this provide only a 64-bit setup, what if the user wants to download 32-bit on windows version < 11 🤔

Reply:
Correct. I'll revise this PR in a bit. There is an active thread here https://github.com/nodejs/nodejs.org/pull/5025#issuecomment-1398683331

Reply:
Ok fixed in [6e21d34](https://github.com/nodejs/nodejs.dev/pull/3163/commits/6e21d343a06a55d8e563f1afd0c5eeb990a33bbf)